int/kpp_dvode.f90

```diff
@@ -14,9 +14,9 @@ MODULE KPP_ROOT_Integrator
 !~~~> Flags to determine if we should call the UPDATE_* routines from within 
 !~~~> the integrator.  If using KPP in an external model, you might want to
 !~~~> disable these calls (via ICNTRL(15)) to avoid excess computations.
-  LOGICAL :: Do_Update_RCONST
-  LOGICAL :: Do_Update_PHOTO
-  LOGICAL :: Do_Update_SUN
+  LOGICAL, PRIVATE :: Do_Update_RCONST
```

Comment:
This (and similar modifications in int/*.f90) are for the toggling of the Update_* functions.

Reply:
I found a bug in our new treatment of Update_* which we need to fix. It's described here:

https://github.com/KineticPreProcessor/KPP/pull/24#issuecomment-1103845571